src/django/api/views/facility/facilities_view_set.py

```diff
@@ -929,6 +929,8 @@ def claim(self, request, pk=None):
                 )
 
             send_claim_facility_confirmation_email(request, facility_claim)
+            facility.updated_at = timezone.now()
```

Comment:
It's better to use one method like `def update_facility_updated_at_field(facility_id)` in all places to have source of truth.

Reply:
replaced